Source/WebKit/Shared/ios/InteractionInformationAtPosition.mm

```diff
@@ -33,6 +33,95 @@
 
 #if PLATFORM(IOS_FAMILY)
 
+InteractionInformationAtPosition::InteractionInformationAtPosition(InteractionInformationRequest&& request, bool canBeValid, std::optional<bool> hitNodeOrWindowHasDoubleClickListener, Selectability&& selectability, bool isSelected, bool prefersDraggingOverTextSelection, bool isNearMarkedText, bool touchCalloutEnabled, bool isLink, bool isImage,
```

Comment:
We can still do without this full constructor, right? What requires this definition now?

Reply:
The generated IPC code tries to construct the object by now passing a  Vector<RetainPtr<DDScannerResult>>&& dataDetectorResults whilst the actual C++ structure still holds a RetainPtr<NSArray> dataDetectorResults;. The constructor is simply converting the type on construction. Is there a more elegant way of achieving this?